include/vrv/devicecontextbase.h

```diff
@@ -102,8 +102,8 @@ class FontInfo {
     {
         m_pointSize = 0;
         m_family = 0; // was wxFONTFAMILY_DEFAULT;
-        m_style = FONTSTYLE_NONE;
-        m_weight = FONTWEIGHT_NONE;
+        m_style = FONTSTYLE_normal;
+        m_weight = FONTWEIGHT_normal;
```

Comment:
Doesn't this mean that `normal` font style / weight will be added to every single text element in the SVG? I think we want to avoid this. @rettinghaus ?

Reply:
Definitely! We have Verovio's "house style" set in the svg style element. So `@fontstyle` should only be added, if it is used in the input file.

Reply:
Ok. I will care about it.